rollup.config.js

```diff
@@ -20,7 +21,11 @@ export default [
       globals,
     },
     external,
-    plugins: [babel(), externalDeps()],
+    plugins: [
+      replace({ 'process.env.NODE_ENV': `"production"`, delimiters: ['', ''] }),
```

Comment:
Would this need to be `development` instead of `production`?

Reply:
my mistake - correcting